### DISCUSSION TITLE
run an ssl server using WiFi with an MKRWiFi1010 (SAMD21) currently using WiFiNINA_Generic

### DISCUSSION THREAD

**Discussion by stefat** [khoih-prog/EthernetWebServer#65] [Q&A]
I don't know if this is the right place for this subject. My apologies if it is not.
Is there any one that could help with this issue ?
Thanks.

**Comment by khoih-prog** [comment]
I think as **MKRWiFi1010** is totally supported by Arduino WiFiNINA library, it's **better and faster** to ask for support there.

The only obstacle I can think is if the `limited processing power` of old SAMD21 can deal with SSL/TLS as a server when dealing with multiple clients simultaneously.

Is it better to move to faster MCU, such as ESP32, Portenta_H7, RP2040W ?

I'm sorry I won't have time to deal with this now. Hopefully someone will chip in and help.
